zerver/webhooks/beeminder/tests.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf -*-
+import time
+from typing import Text
+from zerver.lib.test_classes import WebhookTestCase
+time = (1515759100 - time.time())/3600
+
+class BeeminderHookTests(WebhookTestCase):
+    STREAM_NAME = 'beeminder'
+    URL_TEMPLATE = u"/api/v1/external/beeminder?api_key={api_key}&email=AARON@zulip.com"
+
+    def test_beeminder_derail(self) -> None:
+        expected_subject = u"beekeeper"
+        expected_message = u"Hello **aaron**! I am the Beeminder bot! :octopus: \n You are going to derail \
+        from goal **gainweight** in **{:0.1f} hours** \n You need **+2 in 7 days (60)** to avoid derailing \n * Pledge: **0$** :relieved:".format(time)
+        self.send_and_test_stream_message('derail', expected_subject, expected_message, content_type="application/x-www-form-urlencoded")
```

Comment:
Could you wrap the arguments at 80 characters please? It is easier to read. :)

Reply:
Thanx @eeshangarg for giving it a review...can you please explain this...should i shorten `expected_message`...?

Reply:
@Rishabh570: What I mean is format the arguments over multiple lines like you did for the api_beeminder_webhook function. Does that make sense?

Reply:
Okay...got it..doing it.!